tests/test_path.py

```diff
@@ -103,5 +105,46 @@ def test_search_silent(self):
         target = self.path.search('shape_missing_box.png', silent=True)
         self.assertIsNone(target)
 
+
+class ScopedPathTest(unittest.TestCase):
```

Comment:
Do you mind adding a class docstring here and for the class above? Something like your previous `"""Tests for the chain target (series of steps)."""` for instance.